cypress/e2e/SwipeCarousel.spec.tsx

```diff
@@ -1,32 +1,31 @@
-const CAROUSEL_DEMO = '/carousel-demo';
+import { getCyInIFrame } from '../support/helpers';
 
 describe('SwipeCarousel', () => {
   const WIDTH = Cypress.config('viewportWidth');
   beforeEach(() => {
-    cy.visit(CAROUSEL_DEMO);
+    cy.visit('/');
+    // eslint-disable-next-line cypress/no-unnecessary-waiting
+    cy.wait(500);
```

Comment:
will we have to add this to every test?

Reply:
I think so, it takes some time for loading the storybook component so then cy can reach its content, and I can dive deeper into it sometime

Reply:
hmm. i wonder if maybe `cypress` isn't the right tool for testing with `storybook`.

i only chose it originally because we have more e2e features with `cypress` (as opposed to something like `jest`, which only has JSDom, so it's sometimes hard to test things that rely on scrolling for example)

Reply:
mmm, we can discuss this and 

> @KostiantynPopovych -- i envisioned `/utils` being a top-level, shared utility folder, so I don't think it should be under `/packages` (also, I think only things we actually plan on publishing should be under `/packages`)

got you, I was just thinking that it will be a separate package as well sometime, bu if no - I'll move it back to the top level